.circleci/config.yml

```diff
@@ -165,31 +159,19 @@ jobs:
       - run:
           name: 'Deploy S3 Tag'
           command: |
-            aws s3 sync patternlab/styleguide/public "s3://mayflower.digital.mass.gov/patternlab/v/$CIRCLE_TAG"
+            aws s3 sync packages/patternlab/styleguide/public "s3://mayflower.digital.mass.gov/patternlab/v/$CIRCLE_TAG"
             if [[ "$CIRCLE_TAG" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
               # Use cp instead of sync for root copy.  sync lists all keys and is too slow.
-              aws s3 cp --recursive patternlab/styleguide/public/ "s3://mayflower.digital.mass.gov/patternlab"
+              aws s3 cp --recursive packages/patternlab/styleguide/public/ "s3://mayflower.digital.mass.gov/patternlab"
             fi
 
   react_build_storybook:
-    <<: *react_defaults
+    <<: *patternlab_defaults
```

Comment:
why does this one need to use php?

Reply:
I want to keep all the jobs on the same images as possible, due to how circleci cache and workspace mounting works.

When you save cache, it saves to the path of the working_directory. react_defaults and patternalb_defaults have different paths, so you can't restore cache to both using the cache created by the `build` job. Having all jobs that need cache on the same image and working_directory fixes this issue. Since it's easier to fix anything missing from node vs anything missing from php on circleci, I chose the php image. That image has node available anyway.

Additionally since we now use rush, and rush installs all dependencies for all packages, I can cache on build early.

Reply:
got it, thanks! maybe we should rename the images in the future